docs/upgrade.html

```diff
@@ -19,6 +19,17 @@
 
 <script id="upgrade-template" type="text/x-handlebars-template">
 
+<h4><a id="upgrade_3_9_0" href="#upgrade_3_9_0">Upgrading to 3.9.0 from any version 0.8.x through 3.8.x</a></h4>
+    <h5><a id="upgrade_390_notable" href="#upgrade_390_notable">Notable changes in 3.9.0</a></h5>
+    <ul>
+        <li>In case you run your Kafka clusters with no execution permission for the <code>/tmp</code> partition, Kafka will not work properly. It might either refuse to start or fail
```

Comment:
> It might either refuse to start

@mimaison After your PR (https://github.com/apache/kafka/pull/16782), I guess I can remove this from the notes in trunk. It still would make sense when I port this PR to `3.8`. WDYT?